### PR TITLE
Editorial: fix abbriviations

### PR DESCRIPTION
**Pull request by marcoscaceres**
The following tasks have been completed:

 * [X] Confirmed there are no ReSpec errors/warnings.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/payment-method-basic-card/pull/74.html" title="Last updated on Mar 28, 2019, 5:37 AM UTC (c014095)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-method-basic-card/74/5e28754...c014095.html" title="Last updated on Mar 28, 2019, 5:37 AM UTC (c014095)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -160,9 +158,9 @@ <h2>
       </p>
       <aside class="note" title="Common issuer identification numbers (IIN)">
         <p>
-          Issuer identification numbers (IIN) are maintained by ISO but are not
-          available to the general public. However, Wikipedia maintains a list
-          of the common <a href=
+          <dfn data-abbr="">Issuer identification numbers</dfn> are maintained
```
Comment (ianbjacobs) [inline index.html:161]: Hi @marcoscaceres, any reason you dropped "(IIN)" ?

Comment (ianbjacobs) [reply index.html:161]: Ah, it looks like you might have planned to with data-abbr but the value is nil...whereas below the value is non-nil (for PCI)....

**Review by ianbjacobs — COMMENTED** [review]
See minor comments inline; otherwise fine!

File: index.html (hunk 1 above)
Comment (marcoscaceres) [reply index.html:161]: Gets automatically added :)

**Review by ianbjacobs — APPROVED** [review]

**Pull request merged**